src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,8 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Represents Object in a printable string representation
+ */
+public interface Printable {
+    public abstract String getPrintableString();
```

Comment:
No need for abstract here. As this is an interface, all methods are abstract anyway.